robot-service/src/main/webapp/profile.html

```diff
@@ -0,0 +1,42 @@
+<h2 class="head_m" id="heading">Profile</h2>
+
+<div class="row">
+    <div class="main col-lg-12 profilePanel" id="content_id">
+        <div class="main col-lg-1">
+            <div align="left">
+                <div class="roundedOne" align="left">
+                    <input id="EDUCATION" type="checkbox" onclick="updateNumberOfSelectedCategories()">
+                    <label for="EDUCATION"> &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbspEducation&nbspAnd&nbspScience </label>
+                </div>
+                <div class="roundedOne">
+                    <input id="TRAVEL" type="checkbox" onclick="updateNumberOfSelectedCategories()">
```

Comment:
Don't you like &nbsp; ?? :de: 
This is just 'space' character :smile:
But, we will change this soon. Thanks :+1: